opacus/tests/dp_layers/dp_lstm_test.py

```diff
@@ -69,11 +69,21 @@ def test_lstm(
 
         dp_lstm.load_state_dict(lstm.state_dict())
 
-        x = (
-            torch.randn([batch_size, seq_len, emb_size])
-            if batch_first
-            else torch.randn([seq_len, batch_size, emb_size])
-        )
+        if packed_input_flag == 0:
```

Comment:
It seems like rebase didn't go well - `packed_input_flag` and `_gen_packed_data ` are missing.

Also, given that `_gen_packed_data` is now used in two different tests, it might be a good idea to move it to utils